### DISCUSSION TITLE
Reloading llng_fastcgi-server and nginx

### DISCUSSION THREAD

**Discussion by Trophime** [nfrastack/container-lemonldap#33] [Q&A]
Hi,
I would like to restart/reload the services within the container.
It seems that the standard 'rc-service' is not available in the image.
What is the way to restart the service in a running container?

Best

**Comment by tiredofit** [comment]
I personally just `pkill llng-fastcgi-server` otherwise there are a couple shortcuts:

service_down (s6 servicename) eg `service_down 10-nginx`. Note this doesn't stop the process, it just stops it from restarting when you kill the process manually.

You can do things with the `s6-svc` command that will work against services that are running in `/etc/services.d`
